boards/rakwireless/rak3112/Kconfig.sysbuild

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2025 Kenneth Lu
```

Comment:
Would you mind keeping original Copyright whenever file is copied and add yours if code is changed? Please check all files in the PR.